src/slice.js

```diff
@@ -20,7 +20,7 @@ export function slice(input, starts, sizes, {axes} = {}) {
   const axesLen = axes.length;
   const outputShape = input.shape.slice();
   for (let i = 0; i < axesLen; ++i) {
-    const axis = axes[i] >= 0 ? axes[i] : axes[i] + rank;
+    const axis = axes[i];
```

Comment:
Aah, nice, we can simplify the testing and implementations after Ningxin's removal of the negative axes policy.